modules/n1ql/pages/n1ql-language-reference/advise.adoc

```diff
@@ -375,10 +374,6 @@ WHERE type = "hotel"
 [[pushdown-properties]]
 === Pushdown Properties
 
-ifeval::['{page-component-version}' == '6.6']
-_(Introduced in Couchbase Server 6.6)_
-endif::[]
-
```

Comment:
I also removed all of these `page-component-version` items. Can revert if they're actually needed for something.